SKILL.tsv

```diff
@@ -4047,12 +4047,12 @@ SKILL_20150428_004046	{memo X}Physical/Magic Defense +#{CaptionRatio}# for Shiel
 SKILL_20150428_004047	{memo X}Summon Demon type boss{nl}Demon power +#{CaptionRatio}#% {nl}Additinal Demon damage #{CaptionRatio2}#
 SKILL_20150428_004048	{memo X}Effect applies #{CaptionRatio}# times
 SKILL_20150428_004049	{memo X}Back masking range100{nl}Cannot back mask for 10seconds in target area{nl}Use 1 Piece of Dimension
-SKILL_20150428_004050	{#DD5500}{ol}[Physical] - [Pierce]{/}{/}{nl}Arrow breaks down to multiple pieces and bounces nearby.
+SKILL_20150428_004050	{#DD5500}{ol}[Physical] - [Pierce]{/}{/}{nl}Arrow splits into to multiple pieces and bounces nearby.
```

Comment:
An arrow that splits into multiple pieces that bounces nearby when attacking a target.